src/layers/geojson-layer/geojson-layer.js

```diff
@@ -73,7 +74,7 @@ export default class GeoJsonLayer extends Layer {
 
     this.dataToFeature = [];
     this.registerVisConfig(geojsonVisConfigs);
-    this.getPositionAccessor = () => featureAccessor(this.config.columns);
+    this.getPositionAccessor = dataContainer => featureAccessor(this.config.columns)(dataContainer);
```

Comment:
delete the other `getPositionAccessor` in this layer class? that was an old implementation